util/ticker/ticker.go

```diff
@@ -0,0 +1,33 @@
+package ticker
+
+import (
+	"time"
+)
+
+type Ticker interface {
```

Comment:
None of the existing mock libs that I found (including the one from apimachinery that is used elsewhere) seems to properly mock the `.Reset()` method on a ticker, so I ended up implementing a small shim to let me mock that out.  If/when they support `.Reset`, we should swap to that.